.setup/ansible/roles/submitty_course_add_instructor/handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Reload fpm
+  ansible.builtin.service:
+    name: php8.1-fpm
```

Comment:
You could also move the version to a vars or inventory file so the php version can be changed in one place in the future
`name: "php{{ php_version }}-fpm"`

Reply:
Thanks. Made this update consistent with all roles.